modelforge/potential/models.py

```diff
@@ -90,10 +84,60 @@ def forward(
             - 'd_ij' : torch.Tenso, shape (3, n_pairs)
 
         """
-        pair_indices = self.calculate_neighbors(
+        pair_indices = self.calculate_pairs(
+            atomic_subsystem_indices,
+            only_unique_pairs=self.only_unique_pairs,
+        )
+        r_ij = self.calculate_r_ij(pair_indices, positions)
+
+        return {
+            "pair_indices": pair_indices,
+            "d_ij": self.calculate_d_ij(r_ij),
+            "r_ij": r_ij,
+        }
+
+
+class NeighbourList(PairList):
```

Comment:
I think it would be good to use one version of neighbor/neighbour.  Most of the other places you use the non-british version of neighbor.

Reply:
Yes! Neighbor it is